src/components/20-molecules/top-content-bar/index.scss

```diff
@@ -1,5 +1,95 @@
+$mediaquery-mobile-up: 411px;
+$mediaquery-tablet-up: 991px;
+
 .m-top-content-bar {
   display: block;
+  line-height: 24px;
+
+  .m-top-content-bar__container {
+    line-height: 24px;
+  }
+
+  &.hide {
+    display: none;
+  }
+
+  &.m-top-content-bar__overlay-desktop,
```

Comment:
I understand it better now. But technically, nothing guarantees the overlay property, as seen in this screenshot where I added some CSS to the subheader:
![Screen Shot 2022-01-06 at 09 38 13](https://user-images.githubusercontent.com/46926507/148923954-a486dfd6-0d92-4516-b2b7-02124fbdae2c.png)
:

Reply:
Apart from the problem with the overlay implementation, a more fundamental question is why a PL component should be given the power to suppress another, neighbouring component - doesn't this break the fundamental contract of a "component", to not draw outside of the box it has been given?

Reply:
One could argue that the orchestration of such suppression of neighbouring components belongs one level higher, to page CSS - and that it has no place in component CSS, since arguably a generic top-content-bar component should not encode narrowly specific usage scenarios.